compiler/src/main/kotlin/software/amazon/lastmile/kotlin/inject/anvil/processor/GenerateKotlinInjectComponentProcessor.kt

```diff
@@ -209,4 +218,16 @@ internal class GenerateKotlinInjectComponentProcessor(
     private fun KSValueParameter.requireDelegateName(): String {
         return "${this.requireName()}Delegate"
     }
+
+    private fun KSClassDeclaration.getAccessModifier(): KModifier {
+        return when (getVisibility()) {
+            Visibility.PUBLIC -> PUBLIC
+            Visibility.PRIVATE -> error("Visibility cannot be private")
```

Comment:
Don't throw an error like that, you should point to the symbol for better error messages. I understand that you check for visibility above, but it's still better to follow best practices.